_examples/culley-2016.md

```diff
@@ -1,12 +1,45 @@
+  <p>
+  The authors investigate optimal <a href="../reservoir-operations/">reservoir operations</a> across hydrometeorological conditions to identify a system performance threshold under climate change uncertainty. That is, system operation is formulated as a <a href="../../theory/feedback-control/">feedback control policy</a>, and the overall objective is to find the theoretical upper limit for a water resources system's adaptation to climate change uncertainty, i.e. the <a href="../../theory/maximum-operational-adaptive-capacity">maximum operational adaptive capacity</a> beyond which even optimal operational rules fail to achieve societal objectives for a system of interest.
+  </p>
+  <p>
+  The approach is a variant of the formal method referred to as <a href="../../theory/decision-scaling">"decision scaling"</a>. Testing performance of the optimal feedback control policies across changes in temperature and precipitation is a form of <a href="../../theory/stress-testing/">stress testing</a>, and the identification of failures to meet objectives is a form of <a href="../../theory/vulnerability-analysis/">vulnerability analysis</a>.
+  </p>
+  <p>Feedback control operating rules are modelled using an approximation technique called "Gaussian radial basis functions" (RBFs), describing reservoir releases over time and as a function of lake level. In addition to estimating a historical operating rule, <a href="../../theory/pareto-optimality"> Pareto-optimal </a> RBF policies are identified using evolutionary multiobjective "direct policy search". </p>
+  Methodological steps: <ol>
+  <li> Generation of an exposure space (H): <ul>
+    <li> making incremental changes to hydrological variables' attributes. A n-dimensional exposure space is created, made of selected attributes (h), such as H = [h1,...,hn].  </li>
+    <li> sampling the exposure space using a scaling of historical data to develop perturbed time series for the relevant climatic variables. </li> </ul>
+    <!-- TODO: we may need to provide more detail about perturbation and stochastic generation methods -->
+  <li>Assessment of current system performance for each point of the exposure space:<ul>
+    <li>Running a simulation model with current operation policy with each climate input (see <a href="../../theory/exploratory-modelling">exploratory modelling</a>)</li>
+    <li>Separating "failure" from "success" conditions based on the definition of thresholds for acceptable values. </li></ul>
+  </li>
+  <li>Assessment of the optimised system performance for each point of the exposure space, i.e. with optimal operation rules for each climate input</li>
+  <li> Superimposition of climate projections onto the exposure space for the current and optimised system performance to identify climate conditions and timing for which the system may fail and when to implement management strategies to maximise a system adaptation capacity (see <a href="../../theory/triggers"> triggers</a>). </li>
```

Comment:
Could you confirm that the paper actually identifies triggers? If so, is there a figure that illustrates this?

Reply:
The paper was not identifying 'triggers' explicitly. You could remove the link to triggers.